helm/featurehub/templates/edge-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edge-config
+data:
+  application.properties: |-
+    server.port=8553
+    maxSlots=60
+    update.pool-size=30
```

Comment:
I think you would expect those values in the values.yaml.

Reply:
true